pkg/lib/serverTime.js

```diff
@@ -496,6 +560,20 @@
             </tbody>
         </table>);
 
+    const mode_options = [
+        <SelectOption key="1" value="manual_time">{_("Manually")}</SelectOption>,
+        <SelectOption key="2" isDisabled={!ntp_supported} value="ntp_time">{_("Automatically using NTP")}</SelectOption>
+    ];
+
+    if (custom_ntp.backend)
+        mode_options.push(
+            <SelectOption key="3" isDisabled={!ntp_supported} value="ntp_time_custom">
+                { custom_ntp.backend == "chronyd"
+                    ? _("Automatically using additional NTP servers")
+                    : _("Automatically using specific NTP servers")
```

Comment:
These 5 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18298-20230207-124146-95ee9c44-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)